test/metrics/test_metrics.py

```diff
@@ -126,6 +127,9 @@ def test_negative_log_predictive_density(self):
     def test_mean_standardized_log_loss(self):
         self._test_metric(mean_standardized_log_loss)
 
+    def test_standardized_mean_squared_error(self):
+        self._test_metric(standardized_mean_squared_error)
```

Comment:
Hmm this is a kind of interesting test in that it only executes the metric code, without really checking what the computation does. Not really in scope for this PR but this should probably do something more reasonable...

Reply:
Agreed. Treating this as out of scope for this PR for now, but created an issue for it: #2326 .